enterprise-cfengine-guide/high-availability/installation-guide.markdown

```diff
@@ -318,6 +325,29 @@ The command should return one entry indicating that *node1* is connected to the
     pcs resource master mscfpgsql cfpgsql master-max=1 master-node-max=1 clone-max=2 clone-node-max=1 notify=true
     ```
 
+5. Group previously configured shared IP address and PostgreSQL cluster resource to make sure both will always run on the same host and add migration rules to make sure that resources will be started and stopped in correct order.
+
+    ```bash
+    pcs constraint colocation add cfengine with Master mscfpgsql INFINITY
+    pcs constraint order promote mscfpgsql then start cfengine symmetrical=false score=INFINITY
+    pcs constraint order demote mscfpgsql then stop cfengine symmetrical=false score=0
+    ```
+
+5. Verify that constraints configuration is correct.
```

Comment:
It doesn't matter in markdown, but I will fix this stuff after adding final changes.